fancy_means/fast_means.py

```diff
@@ -0,0 +1,14 @@
+%%cython
```

Comment:
This is a jupyter specific command. You don't actually need to use it outside of jupyter.